docs/Secure-Coding-Guide-for-Python/CWE-664/CWE-197/01/compliant01.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
+# SPDX-License-Identifier: MIT
+""" Compliant Code Example """
+foo = int(round(0.9))
+type(foo)  # class int
+print(foo)  # 0
```

Comment:
why is there a 0?

Reply:
There seems to be a mistake, the 0 should be in the empty comment in noncompliant01.py, while here it should be a 1.